openspec/changes/codebase-import-runtime-hardening/specs/bundle-packaged-resources/spec.md

```
@@ -0,0 +1,17 @@
+# MODIFIED Requirements
+
+### Requirement: Official bundles SHALL ship module-owned resource payloads
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)